wagtail/users/views/users.py

```diff
@@ -362,14 +362,6 @@ class UserViewSet(ModelViewSet):
 
     template_prefix = "wagtailusers/users/"
 
-    def get_common_view_kwargs(self, **kwargs):
-        return super().get_common_view_kwargs(
-            **{
-                "usage_url_name": None,
-                **kwargs,
-            }
-        )
```

Comment:
I misunderstood how the reference index works. I thought if a model isn't registered with the index, we wouldn't track any inbound references as well. Turns out we do, so the usage view can still be useful for the User model despite not being registered to the index, as we can now display any references to a user with `on_delete=PROTECT` on them.